crates/core/src/transcript/mod.rs

```diff
@@ -124,7 +126,15 @@ impl<Challenger_: Default + Challenger> Default for ProverTranscript<Challenger_
 
 impl<Challenger_: Challenger> ProverTranscript<Challenger_> {
 	pub fn finalize(self) -> Vec<u8> {
-		self.combined.buffer.to_vec()
+		let transcript = self.combined.buffer.to_vec();
```

Comment:
@jimpo you asked to add to `into_verifier`, but I moved it to the `finalize` instead to also have the ability to dump examples.